### ISSUE TITLE
"Generic type 'Options<sync>' requires 1 type argument(s)" error on `Sass` type

### ISSUE THREAD

**Issue by DetachHead** [sveltejs/svelte-preprocess#559]
**Describe the bug**
```
> tsc
node_modules/svelte-preprocess/dist/types/options.d.ts:29:33 - error TS2314: Generic type 'Options<sync>' requires 1 type argument(s).

29 export declare type Sass = Omit<SassOptions, 'file' | 'data'> & {
                                   ~~~~~~~~~~~


Found 1 error in node_modules/svelte-preprocess/dist/types/options.d.ts:29
```

**To Reproduce**
```ts
import preprocess from 'svelte-preprocess'
```

**Expected behavior**
no error

**Information about your project:**

- `svelte-preprocess` version 4.10.7

**Additional context**
workaround is to enable `skipLibCheck` in tsconfig, which isn't ideal

**Comment by KotlinIsland** [comment]
I at a loss trying to imagine how a critical type error such as this could make it into production.

**Comment by DetachHead** [comment]
my guess is because this project uses `ts-jest` which has a [critical issue where it skips most type checking](https://github.com/kulshekhar/ts-jest/issues/3859)

**Comment by kaisermann** [comment]
Hey @DetachHead 👋  this will hopefully be fixed in #562

**Comment by DetachHead** [comment]
> I've published `5.0.0@next` can you try with it?

it works, thanks!
